ckanext/validation_schema_generator/plugin.py

```diff
@@ -10,10 +10,7 @@
 from . import helpers
 from .logic import action, auth, validators
 
-if helpers.is_ckan_29():
-    from .flask_plugin import MixinPlugin
-else:
-    from .pylons_plugin import MixinPlugin
+from .flask_plugin import MixinPlugin
```

Comment:
can we merge the flask_plugin into plugin.py so we don't need to Mixin anymore.

Reply:
https://github.com/qld-gov-au/ckanext-validation-schema-generator/pull/31